robowflex_library/include/robowflex_library/planning.h

```diff
@@ -312,6 +312,8 @@ namespace robowflex
             int max_state_sampling_attempts;        ///< Maximum number of attempts to sample a new state.
             int minimum_waypoint_count;             ///< Minimum number of waypoints in output path.
             bool simplify_solutions;                ///< Whether or not planner should simplify solutions.
+            bool hybridize_solutions;               ///< Whether or not planner should hybridize solutions.
+            bool interpolate_solutions;             ///< Whether or not planner should interpolate solutions.
```

Comment:
Shouldn't these also be set as parameters on the server? Does the OMPL plugin understand those?

Reply:
No it does not, you can only modify them through the [ModelBasedPlanningContext](https://github.com/ros-planning/moveit/blob/master/moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/model_based_planning_context.h#L267)
and not through the [ompl_interface](https://github.com/ros-planning/moveit/blob/master/moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/ompl_interface.h#L122) although we should change MoveIt at some point to do so